tests/integrationv2/test_signature_algorithms.py

```diff
@@ -0,0 +1,176 @@
+import copy
+import pytest
+
+from configuration import available_ports, ALL_TEST_CIPHERS, ALL_TEST_CURVES, ALL_TEST_CERTS, PROVIDERS, PROTOCOLS
```

Comment:
`PROVIDERS` and `PROTOCOLS` are unused in this test.

Reply:
Removed